src/components/search/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode, useState } from 'react';
+import styled from 'styled-components';
+import { getSearchResults } from '../../services/product.service';
+import { Alert } from '../general/Alert';
+import { Button } from '../general/button';
+import { Input } from '../general/input';
+
+const StyledButton = styled.div`
+  width: 20%;
+`;
+
+export const SearchInput: React.FC = () => {
+  const [searchTerm, setSearchTerm] = useState('');
+  const handleClick = async () => {
+    try {
+      const searchData = await getSearchResults(searchTerm);
+      console.log(searchData);
+    } catch (err: unknown) {
+      if (err instanceof Error) {
```

Comment:
Nice! Someone paid attention to the presentation :) Very well done

Reply:
Every time you do something that has a chance of going wrong, remember to handle the errors so nothing crashes

Reply:
Thanks!!